pkg/wallet/account.go

```diff
@@ -125,7 +129,7 @@ func (a *Account) Decrypt(passphrase string) error {
 	if a.EncryptedWIF == "" {
 		return errors.New("no encrypted wif in the account")
 	}
-	a.privateKey, err = keys.NEP2Decrypt(a.EncryptedWIF, passphrase)
+	a.privateKey, err = keys.NEP2Decrypt(a.EncryptedWIF, passphrase, a.Scrypt)
```

Comment:
And to `Decrypt`, you either have a wallet you know when you use it or you're using the default settings.